scripts/ioc_deploy.py

```diff
@@ -135,13 +162,25 @@ def get_parser(subparser: bool = False) -> argparse.ArgumentParser:
         type=force_lower,
         help="Select whether to make the deployment permissions read-only (ro) or read-write (rw).",
     )
+    # rebuild_parser doesn't have unique arguments but let's be consistent in our approach
+    rebuild_parser = subparsers.add_parser(
+        REBUILD_CMD,
+        help=(
+            f"Use 'ioc-deploy {REBUILD_CMD}' to help rebuild write-protected releases. "
+            f"See 'ioc-deploy {REBUILD_CMD} --help' for more information."
+        ),
+        description=(
+            "Rebuild a deployment, even if it is write protected. "
+            "This will briefly relax write permissions, run make, and then reapply permission restrictions."
+        ),
+    )
     # shared arguments
-    for parser in main_parser, perms_parser:
+    for parser in main_parser, perms_parser, rebuild_parser:
         parser.add_argument(
             "--name",
             "-n",
             action="store",
-            default="",
+            default=argparse.SUPPRESS,
             help=(
                 "The name of the repository to deploy. This is a required argument. "
```

Comment:
https://docs.python.org/3/library/argparse.html#required

Though python suggests we don't, we can specify this argument as required if we want it to be.

Reply:
Same with `--release`